File: ОП/Lab 1/Lab2 Python/Lab1.py
import math

def func():
    PI = math.pi
    print("Enter radius(R):")
    R = int(input())
    print("Enter height(H):")
    H = int(input())
    V1 = float(R * R * H * PI)
    V2 = float(V1/3)
    print("Volume of the cylinder:")
    print (V1)
    print ("Volume of the cone:")
    print(V2)
    input("Press Enter to continue.")
func()

File: ОП/Lab 1/Lab1 C/Lab1/Lab1/Lab1.cpp
#include <iostream>
#include <math.h>
#include <stdio.h>
using namespace std;

int main()
{
    int R, H;
    double V1, V2;
    const float PI = 3.14;

    //Введення даних
    cout << "Enter the height: ";
    cin >> H;
    cout << "Enter the radius: ";
    cin >> R;
    
    //Розрахунок
    V1 = R * R * H * PI;
    V2 = V1 / 3;
    
    //Виведення даних
    cout << "Volume of the cilinder: " << V1 << endl;
    cout << "Volume of the con: " << V2 << endl;
    system("pause");
    return 0;
}
